### ISSUE TITLE
Enable design mode for WPF designer

### ISSUE THREAD

**Issue by virzak** [mono/SkiaSharp#2110]
**Is your feature request related to a problem? Please describe.**

Need rendering in Design mode

**Describe the solution you'd like**

Removing these two lines should do the trick

https://github.com/mono/SkiaSharp/blob/bcb6526eeeda9f9e4442f6a716d1ad8c8e15e4cd/source/SkiaSharp.Views/SkiaSharp.Views.WPF/SKElement.cs#L49-L50

**Additional context**

There could be an issue with design time when referencing a project, but should work great with nuget package.

https://github.com/dotnet/sdk/issues/8645

**Comment by mgood7123** [comment]
have you tested this and verified that it actually works?

assuming you dont mean Design Mode for Skia itself (eg a UI Designer for Skia)

**Comment by virzak** [comment]
@mgood7123  Yes, I have tested it. Works in Visual Studio that consumes a nuget package.

**Comment by mgood7123** [comment]
Nice
